packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts

```diff
@@ -135,6 +135,48 @@ describe('Routing', () => {
     );
   });
 
+  it('allows controllers to define single value array params via decorators', async () => {
+    class MyController {
+      @get('/greet')
+      greet(
+        @param.array('stringArray', 'query', {type: 'string'})
+        stringArray: string[],
+      ) {
+        return `hello ${stringArray}`;
+      }
+    }
+    const app = givenAnApplication();
+    const server = await givenAServer(app);
+    givenControllerInApp(app, MyController);
+    return (
+      whenIMakeRequestTo(server)
+        .get('/greet?stringArray=world')
+        // Then I get the result `hello world` from the `Method`
+        .expect('hello world')
+    );
+  });
+
+  it('allows controllers to define multi-value array params via decorators', async () => {
+    class MyController {
+      @get('/greet')
+      greet(
+        @param.array('stringArray', 'query', {type: 'string'})
+        stringArray: string[],
+      ) {
+        return `hello ${stringArray}`;
+      }
+    }
+    const app = givenAnApplication();
+    const server = await givenAServer(app);
+    givenControllerInApp(app, MyController);
+    return (
+      whenIMakeRequestTo(server)
+        .get('/greet?stringArray=loopback&stringArray=community')
+        // Then I get the result `hello world` from the `Method`
```

Comment:
@mrmodise the comment is wrong as the result is `'hello loopback,community'` instead of `'hello world'`

Reply:
Cool will clean up. PR is still in draft mode since objective was to validate the condition, as per Raymond's comment below. Thanks